docs/multicluster/upgrade.md

```diff
@@ -37,6 +37,31 @@ for the feature in new version.
 It should have no impact during upgrade to those imported resources like Service, Endpoints
 or AntreaClusterNetworkPolicy.
 
+### Upgrade from a version prior to v1.13
+
+Prior to v1.13, the CRD `ClusterClaim` is used to define both Cluster and ClusterSet IDs
+for a ClusterSet. Since Antrea v1.13, the CRD `ClusterClaim` is removed, a new field `clusterID`
+is added in the CRD `ClusterSet`, and the name of a ClusterSet CR must match the ClusterSet ID.
+
+For any version prior to v1.13, once you finish the upgrade for an existing ClusterSet, you
```

Comment:
Do you mean MC Controller should be upgraded first? But what will happen if the Controller starts with the old CRs?

Reply:
I refined this part, since ClusterSet is updated to v1alpha2, user can update the manifests first, then recreate the ClusterSet. The old CRs will take no affect.